io/driver/driver_bind_test.py

```diff
@@ -45,6 +46,10 @@ def setUp(self):
         self.count = int(self.params.get('count', default=1))
         if not self.pci_devices:
             self.cancel("No pci_adresses Given")
+        smm = SoftwareManager()
+        pkg = "pciutils"
+        if not smm.check_installed(pkg) and not smm.install(pkg):
```

Comment:
new variable can be eliminated it can be like 
if not smm.check_installed('pciutils') and not smm.install('pciutils'):

Reply:
@PraveenPenguin The reason I went with loop earlier, and variable name was,
1. uniformity of code across the tests.
2. not worrying about pep8 in such trivial code paths.

I can go ahead and take care of all of these comments anyways, but it is too much code changes when we add packages in future.

Reply:
@PraveenPenguin Your thoughts here please

Reply:
@narasimhan-v I think we can add logic when it needed

Reply:
@PraveenPenguin Sorry, I did not get which method you are suggesting in your last comment.

Reply:
i am saying loop and other logic we can add when it is need that's what i feel

Reply:
Ok sure